docs/data/data-grid/column-groups/column-groups.md

```diff
@@ -1,18 +1,100 @@
 ---
-title: Data Grid - Column groups 🚧
+title: Data grid - Column groups
 ---
 
-# Data grid - Column groups 🚧
+# Data grid - Column groups
 
 <p class="description">Group your columns.</p>
 
+Grouping columns allows you to have multiple levels of columns in your header and to toggle column groups to show and hide additional columns.
+
+## Define column grouping
+
+You can define the column grouping structure by providing the `columnGroupingModel` prop to the grid.
+This prop receives an array of column groups.
+
+A column group is defined by at least two attributes:
+
+- `groupId`: a string used to identify the group
```

Comment:
Should we warn that `groupId` should be different than columns `field`?
I've noticed an interesting side-effect:
```tsx
{
    groupId: 'firstName',
    description: '',
    children: [{ field: 'firstName' }, { field: 'lastName' }],
  },
```

Clicking on this group header will toggle sorting of `firstName` column, which may be useful in some cases :)

Reply:
Could we say it's a feature? 🤯😄

I will modify the component such that `onClick` do nothing, because later on we could expect at add on click to toggle group expansion, or other action specific to column group